src/control/cmd/dmg/pool.go

```diff
@@ -112,6 +113,21 @@ func (d *PoolOverwriteACLCmd) Execute(args []string) error {
 	return poolOverwriteACL(d.log, d.conns, d.UUID, d.ACLFile)
```

Comment:
I know this has been done historically but probably don't need to separate this into a separate function, can just put the logic in the Execute method